consensus/helpers.go

```diff
@@ -40,17 +38,17 @@ func (m *consensusModule) getQuorumCertificate(height uint64, step typesCons.Hot
 	var pss []*typesCons.PartialSignature
 	for _, msg := range m.messagePool[step] {
 		if msg.GetPartialSignature() == nil {
-			m.nodeLog(typesCons.WarnMissingPartialSig(msg))
+			m.logger.Warn().Msg(typesCons.WarnMissingPartialSig(msg))
```

Comment:
_This is just an example that can be applied to most if not all loglines from `consensus/types/errors.go`_

currently `WarnMissingPartialSig` returns a string with a coded value:
```
func WarnMissingPartialSig(msg *HotstuffMessage) string {
	return fmt.Sprintf("[WARN] No partial signature found for step %s which should not happen...", StepToString[msg.GetStep()])
}
```

But the idea behind structured logs is to move the values into a separate attribute to make it easier to filter/process/index later:
```
m.logger.Warn().Str("step", StepToString[msg.GetStep()]).Msg("No partial signature found which should not happen...")
```

Which will result in json output looking like `{"step": "stepName", "message": "No partial signature found which should not happen..."}`. This allows us to, say, filter by `step = "stepName"` and we could get all log lines related to that step.

Reply:
Updated the loglines :)